flocker/control/test/test_persistence.py

```diff
@@ -249,3 +303,88 @@ def test_complex_keys(self):
                                manifestations={}, paths={},
                                devices={uuid4(): FilePath(b"/tmp")})
         self.assertEqual(node_state, wire_decode(wire_encode(node_state)))
+
+
+class FakeMigration(object):
```

Comment:
This is more like a stub (https://jml.io/pages/test-doubles.html) - specifically, since the only configuration it can deal with is one with a fixed item, `{"configuration: "fake"}` - regardless of the input given.